File: wmcm/lib/models/course.dart
class Course {
  final String id;
  String title;
  String category;
  String description;
  String status;
  int students;
  String lastUpdated;

  Course({
    required this.id,
    required this.title,
    required this.category,
    this.description = '',
    this.status = 'draft',
    this.students = 0,
    required this.lastUpdated,
  });
}

File: wmcW/src/models/Course.js
export class Course {
  constructor(id, title, category, status = 'draft', students = 0, lastUpdated = new Date().toISOString()) {
    this.id = id;
    this.title = title;
    this.category = category;
    this.status = status;
    this.students = students;
    this.lastUpdated = lastUpdated;
  }
}